src/crimson/os/seastore/object_data_handler.h

```diff
@@ -23,6 +23,8 @@ struct ObjectDataBlock : crimson::os::seastore::LogicalCachedExtent {
     : LogicalCachedExtent(std::move(ptr)) {}
   ObjectDataBlock(const ObjectDataBlock &other)
     : LogicalCachedExtent(other) {}
+  ObjectDataBlock(extent_len_t length)
+    : LogicalCachedExtent(length) {}
```

Comment:
this has been defined in https://github.com/aisakaki/ceph/blob/f19de7c77f7d6b783802f6c18e2e9fe106c90243/src/crimson/os/seastore/cached_extent.h#L740

Reply:
This is just a constructor for the base class